admin/test/test_release.py

```diff
@@ -5,8 +5,8 @@
 """
 
 import os
-from unittest import skipUnless, TestCase
-import tempfile
+from twisted.trial.unittest import SynchronousTestCase as TestCase
```

Comment:
Is this "as" just to not have to change many test definitions? Can it just be a find / replace?